### PR TITLE
Update calendar.yml

### PR DESCRIPTION
**Pull request by brad4d**

### CODE REVIEW

File: models/calendar.yml [hunk 1]
```diff
@@ -83,7 +83,17 @@
 
     - 7:45 PM: Speaker TBA
 
-    - 8:00 PM: Speaker TBA
+    - 8:00 PM:
+        type: Talk
+        title: Asynchronous functions
+        person:
```
Comment (billyroh) [inline models/calendar.yml]: You'll need to add `twitter` to this entry. Like:
```yml
person:
  name: Bradford C. Smith
  twitter: bradford_c_smith
```

File: models/calendar.yml [hunk 2]
```diff
@@ -83,7 +83,17 @@
 
     - 7:45 PM: Speaker TBA
 
-    - 8:00 PM: Speaker TBA
+    - 8:00 PM:
+        type: Talk
+        title: Asynchronous functions
```
Comment (billyroh) [inline models/calendar.yml]: You'll also need to add `emoji` too. Like:
```yml
type: Talk
title: Asynchronous functions
emoji: 💯
```

File: models/calendar.yml [hunk 3]
```diff
@@ -83,7 +83,17 @@
...
+        person:
+          name: Bradford C. Smith
+        description: With the `async` and `await` keywords defined in
```
Comment (billyroh) [inline models/calendar.yml]: This is a nit, but it's best to follow this format:
```yml
description: |
  With the `async` and `await`...
```

File: models/calendar.yml (hunk 2 above)
Comment (brad4d) [inline models/calendar.yml]: Done.

File: models/calendar.yml (hunk 1 above)
Comment (brad4d) [inline models/calendar.yml]: Done, but I had to go look up my Twitter ID because I haven't used it in years.
Why is this required?

File: models/calendar.yml (hunk 3 above)
Comment (brad4d) [inline models/calendar.yml]: done

**Comment by chromakode** [comment]
This looks great!

Would you consider adding a little more detail to the title? I'd suggest: "Asynchronous functions with async and await"

**Comment by brad4d** [comment]
@chromakode, I updated the title. PTAL.

**Comment by chromakode** [comment]
Awesome, merged! :raised_hands:
